### DISCUSSION TITLE
Put Sells in lots??

### DISCUSSION THREAD

**Discussion by reefland** [achannarasappa/ticker#173] [Q&A]
Are we supposed to put a sale of a position in the lots? is that just a negative `quantity` with `unit_cost`? Didn't see anything in readme or configuration example about how to handle a sell.

**Comment by achannarasappa** [comment]
To represent a sale, you'd reduce the quantity of the equity in cost basis lots. Lots are intended to represent the current state of your holdings at any given time rather than a records of each transaction.
